doc/apiary.apib

```diff
@@ -0,0 +1,754 @@
+                                "created": "2017-12-20T18:42:21.091596+00:00",
+                                "id": 15,
+                                "label": "led",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "bool"
+                            },
+                            {
+                                "created": "2017-12-20T18:42:21.092641+00:00",
+                                "id": 16,
+                                "label": "fan",
+                                "template_id": "4",
+                                "type": "dynamic",
+                                "value_type": "bool"
+                            }
+                        ],
+                        "created": "2017-12-20T18:42:21.087782+00:00",
+                        "id": 4,
+                        "label": "SensorModel"
+                    }
+                ]
+            }
+
+
+## Template info [/template/{id}]
+Access a specific template
++ parameters
+    + id: `83af` (required, string) - The template ID
+
+### Delete template [DELETE]
+Removes a template. If any device is based on the template being removed, then all its attributes will be also removed.
++ Response 200 (application/json)
+
+            {
+                "removed": {
+                    "attrs": [
+                        {
+                            "created": "2017-12-20T18:42:21.089367+00:00",
+                            "id": 13,
+                            "label": "temperature",
+                            "template_id": "4",
+                            "type": "dynamic",
+                            "value_type": "float"
+                        },
+                        {
+                            "created": "2017-12-20T18:42:21.090601+00:00",
+                            "id": 14,
+                            "label": "pressure",
+                            "template_id": "4",
+                            "type": "dynamic",
+                            "value_type": "float"
+                        },
+                        {
+                            "created": "2017-12-20T18:42:21.091596+00:00",
+                            "id": 15,
+                            "label": "led",
+                            "template_id": "4",
+                            "type": "dynamic",
+                            "value_type": "bool"
+                        },
+                        {
+                            "created": "2017-12-20T18:42:21.092641+00:00",
+                            "id": 16,
+                            "label": "fan",
+                            "template_id": "4",
+                            "type": "dynamic",
+                            "value_type": "bool"
+                        }
+                    ],
+                    "created": "2017-12-20T18:42:21.087782+00:00",
+                    "id": 4,
```

Comment:
The reponse id here also does not match the request example

Reply:
Okok